README.md

```diff
@@ -12,7 +12,6 @@
 - Technologies used: Git, Github, Gitflow, linters, VS code, npm, etc.
 
 
-
 ## Getting Started
```

Comment:
- [ ] It is a requirement that we use the new `Readme` template from here now on, kindly follow this [link to implement](https://github.com/microverseinc/readme-template)